test/Map/UserDrawing.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
```

Comment:
We usually name specs `UserDrawingSpec.js` instead of just `UserDrawing.js`.  It makes it easier to find the right file in the debugger and in editors.